backend/packages/Upgrade/test/integration/Experiment/enrollmentCode/ExperimentExperimentEnrollmentCode.ts

```diff
@@ -1,259 +1,269 @@
-
-//   // mark experiment point for user 2
-//   markedExperimentPoint = await markExperimentPoint(experimentUsers[3].id, experimentName, experimentPoint, condition, new UpgradeLogger());
-//   checkMarkExperimentPointForUser(
-//     markedExperimentPoint,
-//     experimentUsers[3].id,
-//     experimentName,
-//     experimentPoint,
-//     2,
-//     ENROLLMENT_CODE.INCLUDED
-//   );
-
-//   // get all experiment condition for user 2
-//   experimentConditionAssignments = await getAllExperimentCondition(experimentUsers[1].id, new UpgradeLogger());
-//   checkExperimentAssignedIsNotDefault(experimentConditionAssignments, experimentName, experimentPoint);
-
-//   // mark experiment point for user 2
-//   markedExperimentPoint = await markExperimentPoint(experimentUsers[1].id, experimentName, experimentPoint, condition, new UpgradeLogger());
-//   checkMarkExperimentPointForUser(
-//     markedExperimentPoint,
-//     experimentUsers[1].id,
-//     experimentName,
-//     experimentPoint,
-//     4,
-//     ENROLLMENT_CODE.INCLUDED
-//   );
-
-//   // get all experiment condition for user 2
-//   experimentConditionAssignments = await getAllExperimentCondition(experimentUsers[2].id, new UpgradeLogger());
-//   checkExperimentAssignedIsNotDefault(experimentConditionAssignments, experimentName, experimentPoint);
-
-//   // mark experiment point for user 2
-//   markedExperimentPoint = await markExperimentPoint(experimentUsers[2].id, experimentName, experimentPoint, condition, new UpgradeLogger());
-//   checkMarkExperimentPointForUser(
-//     markedExperimentPoint,
-//     experimentUsers[2].id,
-//     experimentName,
-//     experimentPoint,
-//     3,
-//     ENROLLMENT_CODE.INCLUDED
-//   );
-// }
+import { Container } from 'typedi';
+import { ExperimentService } from '../../../../src/api/services/ExperimentService';
+import { UserService } from '../../../../src/api/services/UserService';
+import { systemUser } from '../../mockData/user/index';
+import { individualAssignmentExperimentConsistencyRuleExperiment } from '../../mockData/experiment/index';
+import { getAllExperimentCondition, markExperimentPoint } from '../../utils';
+import {
+  checkMarkExperimentPointForUser,
+  checkExperimentAssignedIsNotDefault,
+} from '../../utils/index';
+import { experimentUsers } from '../../mockData/experimentUsers/index';
+import { EXPERIMENT_STATE } from 'upgrade_types';
+import { UpgradeLogger } from '../../../../src/lib/logger/UpgradeLogger';
+
+export default async function testCase(): Promise<void> {
+  const experimentService = Container.get<ExperimentService>(ExperimentService);
+  const userService = Container.get<UserService>(UserService);
+
+  // creating new user
+  const user = await userService.upsertUser(systemUser as any, new UpgradeLogger());
+
+  // experiment object
+  const experimentObject = individualAssignmentExperimentConsistencyRuleExperiment;
+  experimentObject.partitions = [
```

Comment:
This is the same partition coming from the `individualAssignmentExperimentConsistencyRuleExperiment`. Why are you adding it again?

Reply:
I am setting excludeIfReached as true for these test cases.